### PR TITLE
os/bluestore: add more means to troubleshoot allocator(s)

### PR DESCRIPTION
**Pull request by ifed01**
Signed-off-by: Igor Fedotov <ifedotov@suse.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/os/bluestore/BlueFS.cc [hunk 1]
```diff
@@ -104,16 +104,16 @@ class BlueFS::SocketHook : public AdminSocketHook {
 	   Formatter *f,
 	   std::ostream& errss,
 	   bufferlist& out) override {
-    if (command == "bluestore bluefs available") {
+    if (command == "bluestore bluefs device info") {
       int64_t alloc_size = 0;
```
Comment (aclamk) [inline src/os/bluestore/BlueFS.cc:108]: I like change "available" -> "device info".
You missed to change this also at `register_command`.

**Review by aclamk — CHANGES_REQUESTED** [review]
Approve, just register_command has to be updated.

File: src/os/bluestore/BlueFS.cc (hunk 1 above)
Comment (ifed01) [reply src/os/bluestore/BlueFS.cc:111]: yep, my bad. Fixed now. Thanks!

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-12-04_09:24:21-rados-wip-kefu-testing-2020-12-04-1516-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-12-05_08:42:04-rados-wip-kefu-testing-2020-12-05-1217-distro-basic-smithi/

**Review by aclamk — APPROVED** [review]

**Comment by tchaikov** [comment]
- https://pulpito.ceph.com/kchai-2020-12-06_12:59:57-rados-wip-kefu-testing-2020-12-06-1640-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-12-08_08:10:54-rados-wip-kefu-testing-2020-12-06-1640-distro-basic-smithi/
- https://pulpito.ceph.com/kchai-2020-12-09_01:18:38-rados-wip-kefu-testing-2020-12-08-2025-distro-basic-smithi/

failures are not related.

**Pull request merged**